factories/Grid.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Factories;
+
+use Contracts\Factories\FactoryContract;
+use Faker\Factory;
+
+class Grid implements FactoryContract
+{
+    /**
+     * Construct the factory.
+     *
+     * @param Factory $faker
+     */
+    public function __construct(Factory $faker)
+    {
+        $this->faker = $faker->create();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function create($limit = 1, $flatten = false, $options = [])
+    {
+        for ($i = 1; $i <= $limit; $i++) {
+            $data[$i] = [
+                'relative_url' => '/styleguide/image/550x400?text=550x400%20',
+                'title' => $this->faker->text(20),
+                'link' => $this->faker->url,
+                'description' => '<p>' . $this->faker->text(5) . ' <a href="#">'. $this->faker->sentence(3)  .'</a>',
```

Comment:
Can we make this link a fake link instead of a hash? Looks like this also needs a closing paragraph tag.

Reply:
Got it